porting/npl/riot/include/nimble/nimble_npl_os.h

```diff
@@ -227,7 +228,7 @@ ble_npl_callout_set_arg(struct ble_npl_callout *co, void *arg)
 static inline uint32_t
 ble_npl_time_get(void)
 {
-    return xtimer_now_usec() / 1000;
+    return xtimer_now_usec64() / US_PER_MS;
```

Comment:
just notice something that could be fixed while we are at it: `ble_npl_time_get()` should actually return a value of type `ble_npl_time_t` (see `nimble/include/nimble/nimble_npl.h` line 139), so we could just adapt the function declaration in this PR.

Furthre: it would be cleaner looking if the return value would be explicitly casted to `ble_npl_time_t`
```c
return (ble_npl_time_t)(xtimer_now_usec64() / US_PER_MS);